src/boardlib/api/aurora.py

```diff
@@ -541,16 +550,17 @@ def calculate_tries_total(group):
     return group
 
 
-def logbook_entries(board, username, password, grade_type="font", db_path=None):
-    login_info = login(board, username, password)
-    token = login_info["token"]
-    user_id = login_info["user_id"]
+def logbook_entries(board, username=None, password=None, token=None, user_id=None, grade_type="font", db_path=None):
+    if not (user_id and token): 
+        login_info = login(board, username, password)
+        token = login_info["token"]
+        user_id = login_info["user_id"]
```

Comment:
Instead of this long method signature, let's just replace the `username, password` arguments with `token, user_id` and move the login logic into the `handle_database_command` function

Reply:
see last commit